url-updater/src/main/java/com/devonfw/tools/ide/url/updater/AbstractUrlUpdater.java

```diff
@@ -11,9 +11,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.time.Duration;
 import java.time.Instant;
-import java.util.Collection;
-import java.util.Locale;
-import java.util.Set;
+import java.util.*;
```

Comment:
You are using Intellij what has stupid default settings.
Star-imports are evil and discouraged but Intellij creates them by default.
You need to configure the number of imports per package required for star import to 999.

Reply:
Adjusted now.